softlayer/resource_softlayer_lb_vpx_service.go

```diff
@@ -129,7 +130,20 @@ func resourceSoftLayerLbVpxServiceCreate(d *schema.ResourceData, meta interface{
 
 	log.Printf("[INFO] Creating LoadBalancer Service %s", *lb_services[0].Name)
 
-	successFlag, err := service.Id(nadcId).UpdateLiveLoadBalancer(lbVip)
+	successFlag := true
+	for count := 0; count < 10; count++ {
+		successFlag, err = service.Id(nadcId).UpdateLiveLoadBalancer(lbVip)
+		log.Printf("[INFO] Creating LoadBalancer Service %s successFlag : %t", *lb_services[0].Name, successFlag)
```

Comment:
small suggestion to store the service name early above and reuse throughout for readability
```go
serviceName := *lb_services[0].Name
```